package.json

```diff
@@ -60,5 +60,6 @@
     "typescript": "5.5.3",
     "typescript-eslint": "7.15.0",
     "vitest": "1.6.0"
-  }
+  },
+  "packageManager": "pnpm@9.12.3+sha512.cce0f9de9c5a7c95bef944169cc5dfe8741abfb145078c0d508b868056848a87c81e626246cb60967cbd7fd29a6c062ef73ff840d96b3c86c40ac92cf4a813ee"
```

Comment:
Could revert this change? I remember it was causing some issues, maybe for local links from another package manager

Reply:
Well it's also causing the CI checks to fail, so there's another reason :P